lib/puppet/provider/package/pip.rb

```diff
@@ -227,12 +227,14 @@ def install_options
     join_options(@resource[:install_options])
   end
 
+  # Quoting is required if the path to the pip command contains spaces.
+  # Required for execpipe() but not execute(), and quoting with execute() causes an error.
```

Comment:
nit: I would replace `and` with `because`. Makes more sense when reading (for me at least).

Reply:
Done and force pushed.